techniques/failures/F111.html

```diff
@@ -113,26 +113,26 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
 		
 
 		<section id="description">
-			<h2>Description</h2>
-			<p>The objective of this Failure is to describe situations where speech input users cannot reliably speak the name of a control because although it has a visible label, it lacks an accessible name.</p>
-			<p>When speech input users interact with a web page, they usually speak a command followed by the reference to some visible label (like the text in a button or a link, or the text labelling some input). If the control lacks an accessible name, speech users won't be able to activate the control using its visible label.</p>	
+			<h2>解説</h2>
+			<p>この失敗例の目的は、コントロールに可視ラベルがあるにもかかわらず、アクセシブルな名前 (name) がないため、音声入力の利用者がコントロールの名前 (name) を確実に発音できない状況を説明することである。</p>
+			<p>音声入力の利用者がウェブページを操作する場合、通常はコマンドに続けて、何らかの可視ラベル (ボタン又はリンク内のテキスト、入力のラベルテキストなど) への参照を発話する。コントロールにアクセシブルな名前 (name) がない場合、音声入力の利用者は可視ラベルを使用してコントロールをアクティブ化することができない。</p>	
 		</section>
 		
 		<section id="examples">
-			<h2>Examples</h2>
+			<h2>事例</h2>
       
       <section class="example" id="example-1-a-text-input-with-a-visible-label-but-without-an-accessible-name">
-				<h3>Example 1: A text input with a visible label, but without an accessible name</h3>
-				<p>The text input is preceded by a visible text label "Enter name", but the text is not marked up as a <code>&lt;label&gt;</code> for the input, and there is no alternative way (e.g., <code>aria-label</code>) to provide the input with an accessible name at all.</p>
+				<h3>事例 1: 可視のラベルはあるがアクセシブルな名前 (name) がないテキスト入力</h3>
+				<p>テキスト入力の前には "Enter name" という可視のテキストラベルがあるが、そのテキストは入力の <code>&lt;label&gt;</code> としてマークアップされておらず、入力にアクセシブルな名前を提供するための代替手段（例：<code>aria-label</code>）も存在しない。</p>
 				<pre><code>
 					&lt;p&gt;Enter name&lt;/p&gt;
 					&lt;input type="text"&gt;
         </code></pre>
       </section>
 
 			<section class="example" id="example-2-a-text-input-with-a-visible-label-and-aria-labelledby-pointing-to-a-non-existent-id">
-				<h3>Example 2: A text input with a visible label and aria-labelledby pointing to a non-existent id</h3>
-				<p>The text input is preceded by a visible text label "Enter name". The text's container has an <code>id</code> of <code>nameEntry</code>, but the input has an <code>aria-labelledby</code> referencing a non-existent <code>name-entry</code> <code>id</code>. As a result, the input lacks an accessible name altogether.</p>
+				<h3>事例 2: 可視ラベル及び存在しない ID を指す aria-labelledby を持つテキスト入力</h3>
```

Comment:
これは誤訳っぽいかも。「可視ラベルを伴う入力欄が、存在しない ID を指す aria-labelledby を持つ」かと思います。前のexampleに表現を揃えるならこうでしょうか。ついでにinputは「入力欄」としてみました。

```suggestion
				<h3>事例 2: 可視のラベルがあり、存在しない ID を指す aria-labelledby を持つテキスト入力欄</h3>
```